frontend/apps/filemanager/filemanager.lua

```diff
@@ -73,22 +86,31 @@ function FileManager:init()
     self.banner = FrameContainer:new{
         padding = 0,
         bordersize = 0,
-        VerticalGroup:new{
-            OverlapGroup:new{
-                home_button,
-                VerticalGroup:new{
-                    TextWidget:new{
-                        face = Font:getFace("smalltfont"),
-                        text = self.title,
-                    },
-                    CenterContainer:new{
-                        dimen = { w = Screen:getWidth(), h = nil },
-                        self.path_text,
+        VerticalGroup:new {
+            CenterContainer:new {
+                dimen = { w = Screen:getWidth(), h = nil },
+                HorizontalGroup:new {
+                    home_button,
+                    VerticalGroup:new {
+                        Button:new {
+                            bordersize = 0,
+                            padding = 0,
+                            text_font_bold = false,
+                            text_font_face = "smalltfont",
+                            text_font_size = 24,
```

Comment:
Since that's the smalltfont size, I don't think this should need to be specified?

Reply:
No, we have to specified "text_font_size" because default value is "20"
https://github.com/koreader/koreader/blob/master/frontend/ui/widget/button.lua#L46

Reply:
Alright, lgtm then